components/core/ConfReader.py

```diff
@@ -15,13 +15,14 @@
 
 def check_conf_availability():
     """Checks whether the conf files have been properly set. Exits otherwise"""
-    dlConf = get_conf_reader('dl.conf')
-    emailConf = get_conf_reader('email.conf')
-
-    if any(val == '' for val in dlConf.values()):
-        sys.exit("Please set all the values in dl.conf")
-    if any(val == '' for val in emailConf.values()):
-        sys.exit("Please set all the values in email.conf")
+    conf = get_conf_reader('main.conf')
+
+    if any(val == '' for val in conf['user'].values()):
```

Comment:
Assuming credentials will be stored in the env, make changes to read from the env instead from the conf file. See my comment in #106